src/js/components/List/stories/action.js

```diff
@@ -5,7 +5,25 @@ import { Grommet, Box, List, Menu } from 'grommet';
 import { More } from 'grommet-icons';
 import { grommet } from 'grommet/themes';
 
-import { data } from './data';
+export const locations = [
+  'Boise',
+  'Fort Collins',
+  'Los Gatos',
+  'Palo Alto',
+  'San Francisco',
+];
+
+export const data = [];
+
+for (let i = 0; i < 40; i += 1) {
+  data.push({
+    entry: `entry-${i + 1}`,
+    location: locations[i % locations.length],
+    date: `2018-07-${(i % 30) + 1}`,
+    percent: (i % 11) * 10,
+    paid: ((i + 1) * 17) % 1000,
```

Comment:
I'm not sure the story uses all these fields, can you please clean up the fields that are not relevant or doesn't impact the story? Same for the other stories.